### PR TITLE
Setup to build orb slam 2 as an external project.

### PR DESCRIPTION
**Pull request by alexmillane**
First attempt at building orb slam 2 as an external project.

### CODE REVIEW

**Comment by alexmillane** [comment]
@helenol

File: CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+      cmake .. -DCMAKE_BUILD_TYPE=Release &&
+      # Uncompressing the vocabulary
+      cd ../../../Vocabulary &&
+      tar -xf ../../orb_slam_2_src/Vocabulary/ORBvoc.txt.tar.gz &&
+      # Cofniguring the orb slam 2 build
+      mkdir -p ../build &&
+      cd ../build &&
+      cmake .. -DCMAKE_BUILD_TYPE=Release
+  BUILD_COMMAND 
+      # Building DBoW2
```
Comment (helenol) [inline CMakeLists.txt:35]: thanks for comments :) Makes it better than g2o...

File: CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+      cd ../orb_slam_2_src/Thirdparty/DBoW2/build &&
+      cmake .. -DCMAKE_BUILD_TYPE=Release &&
+      # Configuring g2o
+      mkdir -p ../../g2o/build &&
+      cd ../../g2o/build &&
+      cmake .. -DCMAKE_BUILD_TYPE=Release &&
+      # Uncompressing the vocabulary
+      cd ../../../Vocabulary &&
+      tar -xf ../../orb_slam_2_src/Vocabulary/ORBvoc.txt.tar.gz &&
+      # Cofniguring the orb slam 2 build
```
Comment (helenol) [inline CMakeLists.txt]: Configuring

File: package.xml [hunk 3]
```diff
@@ -0,0 +1,19 @@
+<package>
+  <name>orb_slam_2_catkin</name>
+  <version>0.0.0</version>
+  <description>orb_slam_2_catkin</description>
+  <maintainer email="alex.millane@google.com">Alexander Millane</maintainer>
```
Comment (helenol) [inline package.xml]: uhhh @google.com? You sure? Put your ETH mail here.

**Review by helenol — APPROVED** [review]
LGTM

**Pull request merged**